### DISCUSSION TITLE
Download latest developing build

### DISCUSSION THREAD

**Discussion by chr0mcom** [iNavFlight/inav#6773] [Q&A] [answered]
hi, is it possible to get a build from developing branch to test the H7 support?

**Comment by JulioCesarMatias** [comment]
hello friend, to compile the current development of iNAV you build it on your computer and compile in order to get what you want, if you are not a programmer and you are not able to build iNAV, you will have to wait for future releases.Document to build iNav >>https://github.com/iNavFlight/inav/tree/master/docs/development

Windows 10 >> https://github.com/iNavFlight/inav/blob/master/docs/development/Building%20in%20Windows%2010%20with%20Linux%20Subsystem.md

last step >> https://github.com/iNavFlight/inav/blob/master/docs/development/IDE%20-%20Visual%20Studio%20Code%20with%20Windows%2010.md

**Comment by joe8012** [comment, marked as answer]
You can try this link.  http://seyrsnys.myzen.co.uk/inav_dev/

**Comment by avsaase** [comment]
Looking at #6371, the timers are not implemented yet so there is not much to test in terms of real flight use.
